### ISSUE TITLE
Database Errors

### ISSUE THREAD

**Issue by MkZ0ck** [Samppaa/GTA-RP#32]
Hey I currently have those issues

18:23:50.247 | Error | GameServer | ERROR STARTING RESOURCE GTA-RP System.Reflection.TargetInvocationException: Ein Aufrufziel hat einen Ausnahmefehler verursacht. ---> System.TypeInitializationException: Der Typeninitialisierer für "GTA_RP.Misc.Singleton`1" hat eine Ausnahme verursacht. ---> System.Reflection.TargetInvocationException: Ein Aufrufziel hat einen Ausnahmefehler verursacht. ---> MySql.Data.MySqlClient.MySqlException: Unable to connect to any of the specified MySQL hosts.
   bei MySql.Data.MySqlClient.NativeDriver.Open()
   bei MySql.Data.MySqlClient.Driver.Open()
   bei MySql.Data.MySqlClient.Driver.Create(MySqlConnectionStringBuilder settings)
   bei MySql.Data.MySqlClient.MySqlPool.CreateNewPooledConnection()
   bei MySql.Data.MySqlClient.MySqlPool.GetPooledConnection()
   bei MySql.Data.MySqlClient.MySqlPool.TryToGetDriver()
   bei MySql.Data.MySqlClient.MySqlPool.GetConnection()
   bei MySql.Data.MySqlClient.MySqlConnection.Open()
   bei GTA_RP.DBManager.IsConnect()
   bei GTA_RP.DBManager.SelectQuery(String query, Action`1 code)
   bei GTA_RP.DBManager.IsTableEmpty(String tableName)
   bei GTA_RP.Vehicles.VehicleManager.InitializeVehicleInsertId()
   bei GTA_RP.Vehicles.VehicleManager..ctor()
   --- Ende der internen Ausnahmestapelüberwachung ---
   bei System.RuntimeTypeHandle.CreateInstance(RuntimeType type, Boolean publicOnly, Boolean noCheck, Boolean& canBeCached, RuntimeMethodHandleInternal& ctor, Boolean& bNeedSecurityCheck)
   bei System.RuntimeType.CreateInstanceSlow(Boolean publicOnly, Boolean skipCheckThis, Boolean fillCache, StackCrawlMark& stackMark)
   bei System.Activator.CreateInstance[T]()
   bei GTA_RP.Misc.Singleton`1..cctor()
   --- Ende der internen Ausnahmestapelüberwachung ---
   bei GTA_RP.Events.EventHandler.InitEvents()
   --- Ende der internen Ausnahmestapelüberwachung ---
   bei System.RuntimeTypeHandle.CreateInstance(RuntimeType type, Boolean publicOnly, Boolean noCheck, Boolean& canBeCached, RuntimeMethodHandleInternal& ctor, Boolean& bNeedSecurityCheck)
   bei System.RuntimeType.CreateInstanceSlow(Boolean publicOnly, Boolean skipCheckThis, Boolean fillCache, StackCrawlMark& stackMark)
   bei System.Activator.CreateInstance(Type type, Boolean nonPublic)
   bei System.Activator.CreateInstance(Type type)
   bei GrandTheftMultiplayer.Server.GameServer.<InstantiateScripts>d__133.MoveNext()
   bei System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
   bei System.Collections.Generic.List`1.InsertRange(Int32 index, IEnumerable`1 collection)
   bei GrandTheftMultiplayer.Server.GameServer.StartResource(String resourceName, String father)

I'm not sure what the Problem is, tested multiple logins for the Database and also turned my firewall of to be sure thats not the point. Also took surely the newest sql file.

**Comment by sjlehtonen** [comment]
Is your MySQL service on and are the login details correct? Did you create the Config.ini file with the correct information? It basically says that the script can't connect to the sql database you have.

**Comment by MkZ0ck** [comment]
Yeah MySQL server is turned on. Even tried it with an other server (one local/one external) config.ini is created and configured. As I mentioned above, I turned the firewall of to check if thats the problem with the external database. Logins should also be fine. I thried several times to delete the database structures and create new ones also users. Maybe I give it another try tomorrow. Maybe some highlighting shit I missed.
